jina/executors/indexers/keyvalue.py

```diff
@@ -2,9 +2,11 @@
 __license__ = "Apache-2.0"
 
 import mmap
+import os
 from typing import Iterable, Optional
 
 import numpy as np
+from tensorflow.python.autograph.pyct.anno import keys
```

Comment:
tensorflow is not a basic dependency of us

Reply:
Don't know how that got in there. Some PyCharm auto-import maybe